engli/102_ABC.md

```diff
@@ -65,57 +46,78 @@ which are pronounced as the consecutive vowels with or without a hiatus in betwe
-
-Semivocalic **u** is written as **u** in the end and as **v** in the beginning of a syllable.
-Therefore the **u** in
-**deu**
-(_a god_)
-changes to **v** in
-**devi**
-(_godlike, divine_).
-It is pronounced as the _w_ in _wet_
-but some speakers may pronounce it more like the _v_ in _vet_.
-
-The letters **y** and **v** appear only in the beginning of a syllable
+Pandunia has two semivowels **y** and **v**.
+They appear only in the beginning of a syllable
 and they are always followed by a full vowel.
 
+
 ### Consonants
 
-Pandunia has 18 consonant sounds.
-Each sound is represented by a unique letter.
+Pandunia has 19 consonant sounds.
+They are represented in the writing system by 19 Latin letters and their combinations.
 Most of them are pronounced in roughly the same way as in English.
-So pronunciation of
-**b, d, f, h, j, k, l, m, n, p, t, z**
-is familiar and easy.
-
-The following consonants are pronounced differently in Pandunia compared to English.
-
-- **g** is always hard like in _girl, get_.
-- **r** is preferably trilled like in Scottish English or the rr in Spanish
-  _carro_. Also other pronunciations are acceptable, including the standard
-  English /ɹ/. Note that **r** is never silent!
-- **s** is always voiceless like in _sissy_. **z** is voiced like in _zoo_.
-- **x** is always pronounced like SH in _ship_.
-- **c** is pronounced like CH in _chip_.
-- **n** is pronounced with the tip of the tongue except in combinations **nk** and **ng**,
-  where it is velar /ŋ/ like in _banker_ and _finger_.
-  In the end of a word, the **g** in **ng** can be silent.
-  So the word **pang** can be pronounced /paŋ/ as well as /paŋg/.
-
-Note: **n** is pronounced with the tip of the tongue except in combinations **nk** and **ng**,
-where it is velar /ŋ/ like in _banker_ and _finger_.
-In the end of a word, the **g** in **ng** can be silent.
-So the word **pang** can be pronounced /paŋ/ as well as /paŋg/.
+
+The table below shows how each Pandunia consonant is pronounced
+by using the symbols of the Internation Phonetic Alphabet (IPA)
+as well as pronunciation advise in plain English.
+
+| Pandunia | IPA    | Pronunciation advise |
+|:--------:|:------:|:---------------------|
+| B        | b      | |
+| C        | ts     | Like _ts_ in _bits_ or alternatively like _c_ in _city_. |
+| Ch       | tʃ     | Always like _ch_ in _chat_. |
+| D        | d      | |
+| F        | f      | |
+| G        | g      | Always hard as in _get_. Never soft as in _gel_. |
+| H        | h (x)  | |
+| J        | dʒ     | Always like J in judge_ or the soft _g_ in _gel_. |
+| K        | k      | |
+| L        | l      | |
+| M        | m      | |
+| N        | n (ŋ)  | |
+| P        | p      | |
+| R        | r (ɹ)  | Preferably trilled as in Scottish and Indian English, or rolled as in American English. Never silent! |
+| S        | s      | Always voiceless like _s_ in _sissy_. |
+| Sh       | ʃ      | Like _sh_ in _shop_. |
```

Comment:
Why do you want to change the consonant spellings to "ch" and "sh"? That seems unrelated to analytic Pandunia.